### PR TITLE
turbostat: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by spageektti — APPROVED** [review]

File: pages.es/linux/turbostat.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# turbostat
+
+> Informa de la topología del procesador, frecuencia, temperatura, potencia y estadísticas de inactividad.
+> Más información: <https://manned.org/turbostat.8>.
+
+- Muestra las estadísticas cada 5 segundos:
+
+`sudo turbostat`
+
+- Muestra las estadísticas cada determinado número de segundos:
```
Comment (tricantivu) [inline pages.es/linux/turbostat.md:10]:
```suggestion
- Muestra las estadísticas cada número de segundos:
```

File: pages.es/linux/turbostat.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# turbostat
+
+> Informa de la topología del procesador, frecuencia, temperatura, potencia y estadísticas de inactividad.
+> Más información: <https://manned.org/turbostat.8>.
+
+- Muestra las estadísticas cada 5 segundos:
```
Comment (tricantivu) [inline pages.es/linux/turbostat.md:6]:
```suggestion
- Muestra las estadísticas cada cinco segundos:
```

File: pages.es/linux/turbostat.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`sudo turbostat`
+
+- Muestra las estadísticas cada determinado número de segundos:
+
+`sudo turbostat -i {{n_segundos}}`
+
+- No decodifica ni imprime la información de la cabecera de configuración del sistema:
```
Comment (tricantivu) [inline pages.es/linux/turbostat.md:14]:
```suggestion
- Muestra información sin decodificar ni imprimir la cabecera de configuración del sistema:
```

File: pages.es/linux/turbostat.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`sudo turbostat --quiet`
+
+- Muestra información útil sobre la CPU cada 1 segundo, sin información de cabecera:
```
Comment (tricantivu) [inline pages.es/linux/turbostat.md:18]:
```suggestion
- Muestra información útil sobre el CPU cada segundo, sin información de cabecera:
```

File: pages.es/linux/turbostat.md (hunk 1 above)
Comment (kant) [reply pages.es/linux/turbostat.md:10]: - Muestra las estadística cada un determinado número de segundos:

Comment (tricantivu) [reply pages.es/linux/turbostat.md:10]: The word `estadística` is not in plural, how about `cierto` instead of `determinado` @kant?

Comment (kant) [reply pages.es/linux/turbostat.md:10]: Sorry, there is a typo: `estadísticas` (my bad!) So, imho, it should be:
* Muestra las estadísticas cada un cierto número de segundos:

Comment (tricantivu) [reply pages.es/linux/turbostat.md:10]: Without the article `un` sounds good to me, does it to you?

Comment (kant) [reply pages.es/linux/turbostat.md:10]: Perfect!

Comment (tricantivu) [reply pages.es/linux/turbostat.md]: I'll update the suggestion to match our agreed solution and apply it

Comment (kant) [reply pages.es/linux/turbostat.md]: updated!

**Review by tricantivu — APPROVED** [review]

**Pull request merged**